Assets/Scripts/UnityServices/Lobbies/LobbyServiceFacade.cs

```diff
@@ -77,88 +81,45 @@ public void BeginTracking()
             if (!m_IsTracking)
             {
                 m_IsTracking = true;

```

Comment:
This is more of an inquiry. I notice that some connection states begin the tracking of a lobby, and therefore begin the subscription of lobby events. Since lobby create/join/delete requests are queried here, and now we have lobby callbacks, what prevents us from beginning tracking here (and potentially ending tracking)?

Reply:
You're right, we could remove the BeginTracking stuff and simply do it automatically when joining a lobby. I explored it at first but I decided instead to keep it how it is to keep symmetry with ending tracking that has to be called manually in OfflineState, and to reduce the amount of changes in this PR. If I am to change this, I think I might take the time to refactor a bit of the code around it, i.e. removing the other calls to LobbyServiceFacade in the LobbyUIMediator and doing it internally instead. Thinking back on it, I think it would be worth it and remove some unneeded dependencies

Reply:
Do you have an opinion on this @SamuelBellomo ?

Reply:
Coming back to this, I think that actually this is out of scope for this PR. The goal here is to replace polling with events, and I think it is better to keep this self-contained. I'll create detailed tickets for further improvements so someone could pick it up when they have time.

Reply:
Want to link the ticket? I think that's fair too. I'll approve after!

Reply:
https://jira.unity3d.com/browse/MTT-5544